config/config.yaml

```diff
@@ -2,6 +2,19 @@ apps:
   faucet:
     port: 3000
     useCachingInterceptor: true
+    gatewayUrl: ${GATEWAY_URL}
+    jwtSecret: ''
+    recaptchaSecret: ''
+    faucetAddress: 'erd1nz88q5pevl6up2qxsgpqgc0qmnm93lh888wwwa68kmz363kdwz9q8tnems'
+    faucetPrivateKeyMode: 'mnemonic' # 'mnemonic' or 'pem'
+    faucetMnemonic: 'find filter man earth citizen student grit limit intact unlock soon sell glass arm join mutual burger horror ghost vapor phrase build artefact alley'
```

Comment:
maybe we can remove faucet Mnemonic if this repository will be public

Reply:
it is a new generated mnemonic. I would leave it here for a better example of how to fill config values